### ISSUE TITLE
GUI elements can't be clicked anywhere when adding servers.

### ISSUE THREAD

**Issue by joshualambert** [jimv39/qvcsos#4]
I'm unable to add any servers to connect to, due to the textfields being disabled. Below is a terminal dump:

``` bash
➜  testenterprise git:(develop) ✗ ./client.sh
Listening for transport dt_socket at address: 5006
13:08:43,329 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
13:08:43,329 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
13:08:43,330 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/Users/joshualambert/Projects/qvcsos/testenterprise/testDeploy/qvcse-gui-3.1.0-SNAPSHOT.jar!/logback.xml]
13:08:43,353 |-INFO in ch.qos.logback.core.joran.spi.ConfigurationWatchList@64fb0d4d - URL [jar:file:/Users/joshualambert/Projects/qvcsos/testenterprise/testDeploy/qvcse-gui-3.1.0-SNAPSHOT.jar!/logback.xml] is not of type file
13:08:43,464 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - debug attribute not set
13:08:43,481 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.core.rolling.RollingFileAppender]
13:08:43,494 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [FILE]
13:08:43,578 |-INFO in ch.qos.logback.core.rolling.FixedWindowRollingPolicy@4003334d - Will use zip compression
13:08:43,612 |-INFO in ch.qos.logback.core.joran.action.NestedComplexPropertyIA - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
13:08:43,638 |-INFO in ch.qos.logback.core.rolling.RollingFileAppender[FILE] - Active log file name: logs/qvcs-server.log
13:08:43,638 |-INFO in ch.qos.logback.core.rolling.RollingFileAppender[FILE] - File property is set to [logs/qvcs-server.log]
13:08:43,640 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
13:08:43,641 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [STDOUT]
13:08:43,643 |-INFO in ch.qos.logback.core.joran.action.NestedComplexPropertyIA - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
13:08:43,644 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [com.qumasoft.guitools.qwin.ActivityPaneLoggingHandler]
13:08:43,644 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [GUI]
13:08:43,648 |-ERROR in ch.qos.logback.core.joran.spi.Interpreter@26:18 - no applicable action for [encoder], current ElementPath  is [[configuration][appender][encoder]]
13:08:43,649 |-ERROR in ch.qos.logback.core.joran.spi.Interpreter@27:22 - no applicable action for [pattern], current ElementPath  is [[configuration][appender][encoder][pattern]]
13:08:43,649 |-INFO in ch.qos.logback.classic.joran.action.RootLoggerAction - Setting level of ROOT logger to INFO
13:08:43,649 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [FILE] to Logger[ROOT]
13:08:43,650 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [STDOUT] to Logger[ROOT]
13:08:43,650 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [GUI] to Logger[ROOT]
13:08:43,650 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - End of configuration.
13:08:43,651 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@46ce4210 - Registering current configuration as safe fallback point

View Utility Commands:
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] System properties:
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.runtime.name = Java(TM) SE Runtime Environment
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.boot.library.path = /Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vm.version = 25.40-b25
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] gopherProxySet = false
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vm.vendor = Oracle Corporation
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vendor.url = http://java.oracle.com/
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] path.separator = :
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vm.name = Java HotSpot(TM) 64-Bit Server VM
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] file.encoding.pkg = sun.io
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] user.country = US
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.java.launcher = SUN_STANDARD
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.os.patch.level = unknown
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vm.specification.name = Java Virtual Machine Specification
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] user.dir = /Users/joshualambert/Projects/qvcsos/testenterprise/testDeploy
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.runtime.version = 1.8.0_40-b25
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.awt.graphicsenv = sun.awt.CGraphicsEnvironment
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.endorsed.dirs = /Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/endorsed
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] os.arch = x86_64
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.io.tmpdir = /var/folders/7z/xrjfnrh949v9dmc81n_9crs80000gn/T/
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] line.separator =

2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vm.specification.vendor = Oracle Corporation
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] os.name = Mac OS X
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.jnu.encoding = UTF-8
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.library.path = /Users/joshualambert/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:.
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.awt.enableExtraMouseButtons = true
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.specification.name = Java Platform API Specification
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.class.version = 52.0
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.management.compiler = HotSpot 64-Bit Tiered Compilers
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] os.version = 10.10.2
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] http.nonProxyHosts = local|*.local|169.254/16|*.169.254/16
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] user.home = /Users/joshualambert
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] user.timezone = America/Chicago
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.awt.printerjob = sun.lwawt.macosx.CPrinterJob
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] file.encoding = UTF-8
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.specification.version = 1.8
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.class.path = qvcse-gui-3.1.0-SNAPSHOT.jar
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] user.name = joshualambert
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vm.specification.version = 1.8
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.java.command = qvcse-gui-3.1.0-SNAPSHOT.jar /Users/joshualambert/Projects/qvcsos/testenterprise/testDeploy
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.home = /Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.arch.data.model = 64
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] user.language = en
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.specification.vendor = Oracle Corporation
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] awt.toolkit = sun.lwawt.macosx.LWCToolkit
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vm.info = mixed mode
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.version = 1.8.0_40
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.ext.dirs = /Users/joshualambert/Library/Java/Extensions:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/ext:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.boot.class.path = /Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/resources.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/rt.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/sunrsasign.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/jsse.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/jce.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/charsets.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/lib/jfr.jar:/Library/Java/JavaVirtualMachines/jdk1.8.0_40.jdk/Contents/Home/jre/classes
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vendor = Oracle Corporation
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] file.separator = /
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] java.vendor.url.bug = http://bugreport.sun.com/bugreport/
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.font.fontmanager = sun.font.CFontManager
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.io.unicode.encoding = UnicodeBig
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.cpu.endian = little
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] socksNonProxyHosts = local|*.local|169.254/16|*.169.254/16
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] ftp.nonProxyHosts = local|*.local|169.254/16|*.169.254/16
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] sun.cpu.isalist =
2015-03-05 13:08:44 INFO [AWT-EventQueue-0] QVCS-Enterprise client version: '3.1.0-SNAPSHOT'.
^[[A^[[A^[[A^[[C^[[C                Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
    at com.qumasoft.guitools.qwin.dialog.ParentBranchComboModel.<init>(ParentBranchComboModel.java:33)
    at com.qumasoft.guitools.qwin.dialog.MaintainViewPropertiesDialog.populateComponents(MaintainViewPropertiesDialog.java:499)
    at com.qumasoft.guitools.qwin.dialog.MaintainViewPropertiesDialog.<init>(MaintainViewPropertiesDialog.java:76)
    at com.qumasoft.guitools.qwin.operation.OperationDefineView.executeOperation(OperationDefineView.java:48)
    at com.qumasoft.guitools.qwin.ProjectTreeControl$ActionDefineView.actionPerformed(ProjectTreeControl.java:1060)
    at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
    at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2346)
    at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
    at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
    at javax.swing.AbstractButton.doClick(AbstractButton.java:376)
    at javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:833)
    at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:877)
    at java.awt.Component.processMouseEvent(Component.java:6525)
    at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
    at java.awt.Component.processEvent(Component.java:6290)
    at java.awt.Container.processEvent(Container.java:2234)
    at java.awt.Component.dispatchEventImpl(Component.java:4881)
    at java.awt.Container.dispatchEventImpl(Container.java:2292)
    at java.awt.Component.dispatchEvent(Component.java:4703)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4533)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
    at java.awt.Container.dispatchEventImpl(Container.java:2278)
    at java.awt.Window.dispatchEventImpl(Window.java:2750)
    at java.awt.Component.dispatchEvent(Component.java:4703)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:751)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:702)
    at java.awt.EventQueue$3.run(EventQueue.java:696)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
    at java.awt.EventQueue$4.run(EventQueue.java:724)
    at java.awt.EventQueue$4.run(EventQueue.java:722)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:721)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by joshualambert** [comment]
The disabled fields can be seen here:

![00000131](https://cloud.githubusercontent.com/assets/1421842/6512751/cb0c1bb4-c33b-11e4-859f-2d633ecb0712.png)

**Comment by jimv39** [comment]
This problem seems to be new to java8u40.

Try using java8u31 available here:
http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html#jdk-8u31-oth-JPR

**Comment by jimv39** [comment]
Others are seeing the same problem:
http://stackoverflow.com/questions/28904317/java-8u40-on-os-x-yosemite

**Comment by jimv39** [comment]
A workaround is to remove the splash screen:
1. Edit the qvcse-gui/pom.xml to comment out the <manifestEntries> (at line 29):
                        &lt;!--
                        &lt;manifestEntries&gt;
                            &lt;SplashScreen-Image&gt;images/qvcs_splash.jpg&lt;/SplashScreen-Image&gt;
                        &lt;/manifestEntries&gt;
                        --&gt;
2. Rebuild everything: mvn clean install -DskipTests
3. cd to testenterprise, and ./updateJars.sh
4. Start the server: ./server.sh
5. Start the client: ./client.sh

**Comment by joshualambert** [comment]
Checking this now against your latest develop branch to see if the issue is corrected now. TY so much for your fast response. This software is used at the office to do version control for some old Oracle Forms 6i and 10g applications, so I needed to be able to download copies of the forms for conversion to Oracle Apex. I got Oracle Forms working on OSX Yosemite with Mac, so a working QVCS was the only thing left to get working. You rock! :)
